src/lib/Scenes/Artwork/Components/ArtworkDetails.tsx

```diff
@@ -17,17 +17,19 @@ export class ArtworkDetails extends React.Component<ArtworkDetailsProps, Artwork
 
   render() {
     const listItems = [
-      { title: "Medium", value: this.props.artwork.medium },
+      { title: "Medium", value: this.props.artwork.category },
       {
         title: "Condition",
         value: this.props.artwork.conditionDescription ? this.props.artwork.conditionDescription.label : null,
       },
       { title: "Signature", value: this.props.artwork.signature },
       {
         title: "Certificate of Authenticity",
-        value: this.props.artwork.certificateOfAuthenticity ? "Included" : "Not Included",
+        value: this.props.artwork.certificateOfAuthenticity
+          ? this.props.artwork.certificateOfAuthenticity.details
+          : null,
       },
-      { title: "Frame", value: this.props.artwork.framed ? "Included" : "Not Included" },
+      { title: "Frame", value: this.props.artwork.framed ? this.props.artwork.framed.details : null },
```

Comment:
This could be further shortened to the following:

```suggestion
      { title: "Frame", value: this.props.artwork.framed && this.props.artwork.framed.details },
      { title: "Frame", value: this.props.artwork.framed ? this.props.artwork.framed.details : null },
```

Which evaluates to the same thing.